pages/desktop/details.py

```diff
@@ -108,10 +106,7 @@ def _page_title(self):
     def title(self):
         base = self.selenium.find_element(*self._title_locator).text
         '''base = "firebug 1.8.9" we will have to remove version number for it'''
-        if "Themes" in self.selenium.find_element(*self._breadcrumb_locator).text:
```

Comment:
Note: not sure the impact (in particular, loss of coverage for Themes) that this removal gives us, but it fixes the test - please review carefully :-)